src/app-layer-htp.c

```diff
@@ -6406,357 +6406,6 @@ static int HTPParserTest16(void)
     return result;
 }
 
-/** \test CONNECT with plain text HTTP being tunneled */
```

Comment:
Can you put these removals in their own commit?

Reply:
Sure